pymilvus/bulk_writer/buffer.py

```diff
@@ -290,49 +289,62 @@ def _persist_parquet(self, local_path: str, **kwargs):
 
     def _persist_csv(self, local_path: str, **kwargs):
         sep = self._config.get("sep", ",")
+        # nullkey is not supported in csv now
```

Comment:
Since CSV bulk insert has already supported nulls, can we add null support here?